packages/select/src/stories/tw-ui-select-theme.config.ts

```diff
@@ -0,0 +1,14 @@
+import { TailwindVariants } from '@unicodernsui/custom-theme'
```

Comment:
this is autogenerated, if you don't plan to apply tailwind soon I recommend you remove it